snoop/data/admin.py

```diff
@@ -1296,7 +1296,7 @@ def get_admin_links():
 
 DEFAULT_ADMIN_NAME = '_default'
 sites = {}
-for collection in collections.ALL.values():
+for collection in collections.get_all(static=True):
```

Comment:
this `static=` arg should be called `static_only` because setting it to False still gives you static entries